internal/instance/resource_instance.go

```diff
@@ -258,17 +290,7 @@ func (r InstanceResource) Schema(_ context.Context, _ resource.SchemaRequest, re
 				},
 				Validators: []validator.String{
 					stringvalidator.LengthAtLeast(1),
-					stringvalidator.ConflictsWith(
-						path.Expressions{
-							path.MatchRoot("description"),
-							path.MatchRoot("type"),
-							path.MatchRoot("ephemeral"),
-							path.MatchRoot("profiles"),
-							path.MatchRoot("file"),
```

Comment:
For any reviewer, `file` was added as mutually exclusive with `source_file` in #172 and then moved here in #181, but I don't think it should be. File uploads are only applied after an instance has been created, and the source of the instance doesn't matter for that. I've therefore removed this restriction entirely.

@breml, is there any particular reason why you added it as mutually exclusive in #172?